<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>嘉義大學_躲避車遊戲</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/src/icon.ico</normaloff>:/src/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>4800</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/map.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="car">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/car.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/game_name.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start_game">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/src/start_btn.png</normaloff>:/src/start_btn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>行駛距離：0公尺</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_clock">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="block01">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block02">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block03">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block04">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block05">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="block06">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>420</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="image.qrc">:/src/item1.png</pixmap>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>block06</zorder>
   <zorder>block05</zorder>
   <zorder>block04</zorder>
   <zorder>block03</zorder>
   <zorder>block02</zorder>
   <zorder>block01</zorder>
   <zorder>car</zorder>
   <zorder>label_title</zorder>
   <zorder>button_start_game</zorder>
   <zorder>label</zorder>
   <zorder>lcd_clock</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>選單</string>
    </property>
    <addaction name="actionstart_game"/>
    <addaction name="actionclose_game"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionstart_game">
   <property name="text">
    <string>開始遊戲</string>
   </property>
  </action>
  <action name="actionclose_game">
   <property name="text">
    <string>結束遊戲</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
